package.json

```diff
@@ -117,8 +117,7 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.8.4",
-    "@babel/core": "7.10.5",
-    "@babel/parser": "7.10.5",
+    "@babel/core": "^7.10.5",
```

Comment:
unlocked this, the was locked to avoid a bug with the old react doc gen version in dev site v6